requirements.txt

```diff
@@ -10,7 +10,7 @@ urllib3==1.24.2
 xmltodict
 git+https://github.com/alliance-genome/agr_genedescriptions.git@v3.2.0#egg=genedescriptions
 git+https://github.com/alliance-genome/agr_file_generator.git@8b47011
-ontobio==1.20.0
+git+https://github.com/valearna/ontobio
```

Comment:
I think we should use canonical versions in PyPi. If there is some change you need on ontobio make a PR and we can discuss merging it! cc @valearna @sierra-moxon @kltm

Reply:
Thanks @cmungall, I inserted a forked version of ontobio because we needed a quick fix for the upcoming release, but I will open a PR on ontobio.